routes/api/homepage.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const passport = require('passport');
+const Post = require('../../models/Posts');
+const User = require('../../models/User');
+
+router.get('/home', passport.authenticate('jwt', { session: false }), (req, res) => {
+  var userFollowing = [];
+  var myPost = mongoose.Types.ObjectId(req.user.id);//get logged in user id to get post
+  
+  User.findById(req.user.id, ('following'))
+  .populate('following.user').lean()
+  .then(following => {
+    if (following) {
```

Comment:
no need of if condition, because "following" would always be truthy(req.user.id)